### ISSUE TITLE
Is there any button/function/script to delete automatically songs from music directory to free up space?

### ISSUE THREAD

**Issue by ik3r4b** [raveberry/raveberry#129]
Thank you in advance!

**Comment by raveberry** [comment]
No, there is (currently) not. It's quite possible that a few large files take up most of the space. Go into the music directory and list large files with `find ./ -size +10M`. You can delete them with `find ./ -size +10M -delete`.
You could also list the oldest files with `find -type f -printf '%T+ %p\n' | sort | head -n 10` and delete those.
